examples/app-with-backend/tests/appCallingBackend.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from "./fixtures";
+
+test("page one should successfully call backend and display reply", async ({
+  pageOne,
+  page,
+}) => {
+  await pageOne.goto();
```

Comment:
Would be good to move the `goto` method call to the fixture (that's what we do in all other page fixtures)